fendermint/vm/interpreter/src/genesis.rs

```diff
@@ -0,0 +1,795 @@
+// Copyright 2022-2024 Protocol Labs
```

Comment:
Is it still strictly necessary to keep this inside the interpreter? Would it be acceptable to move it to `fendermint_vm_genesis` crate?

Reply:
Good question.